scheduler/src/cook/kubernetes/api.clj

```diff
@@ -1016,7 +1026,9 @@
             (.setReadinessProbe container readiness-probe)))
 
           ; resources
-          (set-mem-cpu-resources resources memory-request memory-limit cpu-request cpu-limit)
+          (if (get (config/kubernetes) :set-memory-limit? true)
```

Comment:
revert this. we actually always want limit for init container and sidecar

Reply:
done